nixos/modules/services/web-apps/pingvin-share.md

```diff
@@ -0,0 +1,86 @@
+# Pingvin Share {#module-services-pingvin-share}
+
+A self-hosted file sharing platform and an alternative for WeTransfer.
+
+## Configuration {#module-services-pingvin-share-basic-usage}
+
+By default, the module will execute Pingvin Share backend and frontend on the ports 8080 and 3000.
+
+I will run two systemd services named `pingvin-share-backend` and `pingvin-share-frontend` in the data directory `/var/lib/pingvin-share`.
+
+Here is a basic configuration:
+
+```nix
+{ ... }:
+
+{
+  services-pingvin-share = {
+    enable = true;
+
+    openFirewall = true;
+
+    settings.dataDir = "/var/lib/data/share";
```

Comment:
The default works just as good and we shouldn't create the impression here, that people need to set the data directory

```suggestion
```